### ISSUE TITLE
Dirichlet distribution with int parameters crashes ipython kernel

### ISSUE THREAD

**Issue by amcknight** [pymc-devs/pymc#886]
```
import numpy as np
import pymc3 as pm
with pm.Model():
    p = pm.Dirichlet('p', np.ones(2, dtype=int), shape=2)
```

Setting dtype to 'int' crashes ipython notebook. This happens with other shapes too.
Setting to 'float' works as intended and setting to 'object' throws an AsTensorError, which is ok. (I didn't try other types.)

**Comment by fonnesbeck** [comment]
I cannot replicate this here.  Can you provide some details on your configuration?

``` python
In [1]: import numpy as np

In [2]: import pymc3 as pm

In [3]: with pm.Model():
   ...:         p = pm.Dirichlet('p', np.ones(2, dtype=int), shape=2)
   ...:     

In [4]: 
```

**Comment by amcknight** [comment]
Thanks for getting back to me so quickly. :)
Here's my conda packages:

```
(pymc3)Andrews-MacBook-Air:~ andrew$ conda list
# packages in environment at /Users/andrew/anaconda/envs/pymc3:
#
appnope                   0.1.0                    py27_0  
backports.ssl-match-hostname 3.4.0.2                   <pip>
cycler                    0.9.0                    py27_0  
decorator                 4.0.4                    py27_0  
enum34                    1.0.4                     <pip>
freetype                  2.5.5                         0  
ipykernel                 4.1.1                    py27_0  
ipython                   4.0.0                    py27_1  
ipython-genutils          0.1.0                     <pip>
ipython_genutils          0.1.0                    py27_0  
ipywidgets                4.1.0                    py27_0  
jinja2                    2.8                      py27_0  
jsonschema                2.4.0                    py27_0  
jupyter                   1.0.0                    py27_0  
jupyter-client            4.1.1                     <pip>
jupyter-console           4.0.3                     <pip>
jupyter-core              4.0.6                     <pip>
jupyter_client            4.1.1                    py27_0  
jupyter_console           4.0.3                    py27_0  
jupyter_core              4.0.6                    py27_0  
libpng                    1.6.17                        0  
markupsafe                0.23                     py27_0  
matplotlib                1.5.0               np110py27_0  
mistune                   0.7.1                    py27_0  
nbconvert                 4.0.0                    py27_0  
nbformat                  4.0.1                    py27_0  
notebook                  4.0.6                    py27_0  
numpy                     1.10.1                   py27_0  
openssl                   1.0.2d                        0  
pandas                    0.17.1              np110py27_0  
path.py                   8.1.2                    py27_0  
patsy                     0.4.0               np110py27_0  
pexpect                   3.3                      py27_0  
pickleshare               0.5                      py27_0  
pip                       7.1.2                    py27_0  
ptyprocess                0.5                      py27_0  
pygments                  2.0.2                    py27_0  
pymc3                     3.0                       <pip>
pyparsing                 2.0.3                    py27_0  
pyqt                      4.11.4                   py27_0  
python                    2.7.10                        2  
python-dateutil           2.4.2                    py27_0  
python.app                1.2                      py27_4  
pytz                      2015.7                   py27_0  
pyzmq                     14.7.0                   py27_1  
qt                        4.8.7                         1  
qtconsole                 4.1.0                    py27_0  
readline                  6.2                           2  
scikit-learn              0.17                np110py27_1  
scipy                     0.16.0              np110py27_1  
seaborn                   0.6.0               np110py27_0  
setuptools                18.5                     py27_0  
simplegeneric             0.8.1                    py27_0  
sip                       4.16.9                   py27_0  
six                       1.10.0                   py27_0  
sqlite                    3.8.4.1                       1  
ssl_match_hostname        3.4.0.2                  py27_0  
terminado                 0.5                      py27_1  
theano                    0.7.0                     <pip>
tk                        8.5.18                        0  
tornado                   4.2.1                    py27_1  
traitlets                 4.0.0                    py27_0  
wheel                     0.26.0                   py27_1  
zeromq                    4.1.3                         0  
zlib                      1.2.8                         0
```

Tell me if there's any other config info you want

**Comment by fonnesbeck** [comment]
Ok, Python 2.7. I am running 3.5 so I will see if I can replicate under Py2. 

In the meantime, you can try installing Theano master from pip. Many issues have been fixed since 0.7. 

```
pip install -U git+https://github.com/Theano/Theano.git
```

**Comment by fonnesbeck** [comment]
I ran this successfully with Python 2.7.6 just now, so closing.
